news/2021-08-21-new-featured-artist-wiklund.md

```diff
@@ -0,0 +1,77 @@
+
+We've snagged **10** classics for mappers to dive into, including everything from **Wiklund**'s nostalgic album *Wing Beats* — a favourite of mappers from the olden days, and [one of osu!'s first album-focused beatmap projects](https://osu.ppy.sh/community/forums/topics/17053). Check it all out from [**Wiklund**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/192)!
+
+Experience some of **Wiklund**'s music for yourself with this 2009 map of [*Billy Boogie*](https://osu.ppy.sh/beatmapsets/9040) as seen through the display that 100% of osu! players were using a decade ago. Once you're done with that, maybe explore the rest of these previews too:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/192/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Billy Boogie
+
+Check out [the beatmap from the video above](https://osu.ppy.sh/beatmapsets/9040) hosted by [yeahyeahyeahhh](https://osu.ppy.sh/users/58042) and [this other beatmap](https://osu.ppy.sh/beatmapsets/22082) hosted by [IceBeam](https://osu.ppy.sh/users/208440)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Whip the Blip
+
+Play through [another classic beatmap](https://osu.ppy.sh/beatmapsets/8299) hosted by [Ultimate](https://osu.ppy.sh/users/53940)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Joy of Living
+
+Try out [this beatmap](https://osu.ppy.sh/beatmapsets/9197) by [0_o](https://osu.ppy.sh/users/56708)! People had some interesting usernames 10 years ago.
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### The Digital Force
+
+Enjoy [yet another beatmap](https://osu.ppy.sh/beatmapsets/11153) by [Metroid](https://osu.ppy.sh/users/10542)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Wing Beats
+
+Have fun with [the first ranked **Wiklund** map](https://osu.ppy.sh/beatmapsets/6499) hosted by [v0xtrot](https://osu.ppy.sh/users/52832)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Crystal Dimension
+
+Last but not least, check out [this beatmap](https://osu.ppy.sh/beatmapsets/11100) hosted by [GladiOol](https://osu.ppy.sh/users/23326)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+---
+
+Surprising absolutely nobody, even more of **Wiklund**'s music can be found on [their Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/192)! Will you be the first person to make a dissonant slider-heavy tech map over retro chiptune music? Nobody's asking for it, but we'll wish you luck anyway.
```

Comment:
any idea why it's written "their" instead of "his"?
check the line 11

Reply:
fixed this